.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,9 @@ Fixes #
 
 <!-- Please describe your changes. -->
 
+## Use of AI Tools
+
+<!-- Please disclose whether AI tooling was used to author the code in the pull request. To what extent was AI used? Was it used as a code reviewer, or was it used to "vibe code" everything? It is your responsibility to review and take responsibility for what AI generates. For more, see CONTRIBUTING.md which includes instructions for how to provide instructions to AI agents. -->
```

Comment:
If you apply https://github.com/WordPress/performance/pull/2193/files#r2389568757, it should probably be put here too :)

Reply:
Done in 1fa61f57

Reply:
Maybe this was a misunderstanding - the copy here is still the old one, it needs to be aligned with what's in `CONTRIBUTING.md`.

Reply:
Humm. I seemed to have missed making the change I imagined myself making! Now addressed in 688a21c62a4c63806fe24e37a2dcb84f5b11de76